reconfiguration/cmf/concord.cmf

```diff
@@ -55,105 +55,112 @@ Msg PruneStatus 18 {
     uint64 sender
     bool in_progress
     uint64 last_pruned_block
+    string configuration_json
 }
 
-Msg GetVersionCommand 19 {
+Msg PruneSwitchModeRequest 19 {
+    uint64 sender_id
+    string configuration_json
+}
+
+Msg GetVersionCommand 20 {
   bytes place_holder
 }
 
-Msg InstallCommand 20 {
+Msg InstallCommand 21 {
     string version
     bool bft_support
 }
 
-Msg InstallStatusCommand 21 {
+Msg InstallStatusCommand 22 {
     optional string version
 }
 
-Msg InstallStatusResponse 22 {
+Msg InstallStatusResponse 23 {
     string version
     bool in_progress
     bool install_completed
 }
 
-Msg GetVersionResponse 23 {
+Msg GetVersionResponse 24 {
```

Comment:
What's the impact of shifting all the message id-s? Doesn't it break backwards-compatibility?

Reply:
AFAIK, you should add new messages at the end, without changing existing IDs.

Reply:
Done